lib/tools/broccoli/node-modules-dest-copy.ts

```diff
@@ -69,7 +70,7 @@ export class DestCopy implements IBroccoliPlugin {
 				});
 			}
 		});
-		if (!_.isEmpty(this.dependencies)) {
+		if (!_.isEmpty(this.dependencies) && !this.$config.debugLivesync) {
```

Comment:
I believe that one should give suggestions only when he understands the specific issue in detail and when he has something to offer.